source/tyr/tests/integration/instance_test.py

```diff
@@ -173,6 +180,7 @@ def test_update_instances(create_instance):
     assert resp['poi_dataset'] == 'priv.poi_dataset'
     assert resp['max_taxi_duration_to_pt'] == 200
     assert resp['max_car_no_park_direct_path_duration'] == 555
+    assert resp['max_bike_direct_path_distance'] == 11000
```

Comment:
It's better to test all the attributes you added

Reply:
yeah I planned to add another commit with tests